php-classes/Slate/Courses/SectionParticipantsRequestHandler.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
@BillClinton this override needs to be deleted now that slate#339 is merged and this branch has been rebased on top of upgrading slate to the release that includes it

Reply:
@BillClinton i got it